### PR TITLE
feat(dependency:install): add release def filter

### LINKED ISSUE #55: Enhancement: dependency:install add a filter based on releasedefinition
As (almost) all commands allow to filter on releasedefinition file, It will be helpfull to have this filter on dependency:install too

recreating from https://github.com/dxatscale/sfpowerscripts/issues/1445, as I thinks it's still pertinent

### PR DESCRIPTION
**Pull request by alanjaouen**
resolve #55 

#### Checklist
All items have to be completed before a PR is merged

- [x] Adhere to [Contribution Guidelines](https://docs.flxbl.io/about-us/contributing-to-fxlbl)
- [x] Updates to Decision Records considered?
- [x] Updates to documentation at [flxbl-sfp Guide](https://github.com/flxbl-io/docs-sfp) considered?
- [x] Tested changes?
- [x] Unit Tests new and existing passing locally?

### CODE REVIEW

**Review by dieffrei — APPROVED** [review]
Awesome! Thank you for the contribution @alanjaouen!!

File: src/commands/dependency/install.ts [hunk 1]
```diff
@@ -43,10 +47,17 @@ export default class Install extends SfpCommand {
             ProjectConfig.getSFDXProjectConfig(null),
             this.flags.installationkeys
         );
-        let externalPackage2s = await externalPackageResolver.resolveExternalPackage2DependenciesToVersions();
+
+        let packages = null;
+        if (this.flags.releasefile) {
```
Comment (dieffrei) [inline src/commands/dependency/install.ts:52]: the flag name is "releaseconfig", however is read as "filename"

**Review by dieffrei — CHANGES_REQUESTED** [review]

File: src/commands/dependency/install.ts (hunk 1 above)
Comment (alanjaouen) [reply src/commands/dependency/install.ts]: indeed, missed it when I unified option name with the rest of sfp 😬

**Comment by dieffrei** [comment]
@alanjaouen thank you again for your contribution. I had to make a few changes in order we can get it merged.
I will follow up your changes on a new PR (with your changes). #108

**Pull request closed without merging**